model.js

```diff
@@ -48,3 +48,25 @@ exports.returnReviewComments = (reviewId) => {
     return Promise.reject({ status: 400, msg: err });
   }
 };
+
+exports.postNewComment = (params, reviewId) => {
```

Comment:
Could rename function to something a bit more obviously different to the one in the model - best practise is to change the verb i.e controller is called postReviewComment and the models called insertReviewComment 💻